lib/views/search/search_view.dart

```diff
@@ -15,9 +20,14 @@ class SearchView extends StatelessWidget {
       resizeToAvoidBottomInset: false,
       body: Column(
         children: [
-          Expanded(
-            child: SearchResultWidget(),
-          ),
+          if (searchQuery.query.isEmpty)
+            Expanded(
+              child: SearchInitialPageWidget(),
+            ),
+          if (searchQuery.query.isNotEmpty)
```

Comment:
Why not else?

Reply:
Removed the above condition and instead of this I wrote a ternary condition.

Reply:
`  Expanded(
            child: searchQuery.query.isEmpty
                ? SearchInitialPageWidget()
                : SearchResultWidget(),
          )`